src/packages/examples-ext/processors/exampleDependenciesBuilder.js

```diff
@@ -36,15 +39,25 @@ module.exports = function exampleDependenciesBuilder (readFilesProcessor, log, g
 
       // traverse deployments, check project dependencies and add local file
       // dependencies to the list of generated documents
-      generateExamplesProcessor.deployments.forEach(function(deployment) {
```

Comment:
considering this and next code, I'd prefer to rewrite it in clearer way of map/reduce chain like the following
```js
$process: function (docs) {
      // probably it worth to use timestamps to generate unique deployment names
      const nameMaker = makeUniqueName.bind(null, []);

      // traverse deployments, check project dependencies and add local file
      // dependencies to the list of generated documents
      generateExamplesProcessor.deployments.map((deployment => {
        const name = nameMaker(deployment.name);
        const processor = processDependency.bind(null, name);
        const commonFiles = deployment.examples && deployment.examples.commonFiles || {};

        return (commonFiles.scripts || [])
          .concat(commonFiles.stylesheets || [])
          .concat(ADDITIONS)  // here goes ['/bower_components/iframe-resizer/js/iframeResizer.contentWindow.min.js']
          .reduce(processor, docs);
      })
}
// change signature (1st and 2nd params)
function processDependency (name, docs, item, index, arr) {
```